robot: 
  name: "franka"
topic:
  sub:
    cmd_pose:
      name: /arm/command/pose
      queue: 1
    cmd_tool_orientation:
      name: /tool/command/orientation
      queue: 1
    cmd_delta_pose: 
      name: /arm/command/delta_pose
      queue: 1  
  pub:
    display_trajectory:
      name: /move_group/display_planned_path
      queue: 1
    current_pose:
      name: /arm/state/pose
srv: 
  disable_collision: 
    name: /move_group/disable_collision
  add_collision: 
    name: "/move_group/add_collision"
  start_position_ctl: 
    name: "/start_position_ctl"
  start_joint_traj_ctl: 
    name: "/start_joint_traj_ctl"
  start_joint_group_ctl: 
    name: "/start_joint_group_ctl"
    


